### PR TITLE
updated copy and some styles

### PR DESCRIPTION
**Pull request by mchristineanderson**
I made a few changes: 

- copy pass on intro block
- changed some styles and spacing on that intro block
- added in two images under greencrate
- removed an image from Felt and changed the ordering a little
- added in a cover image and meta data for social sharing. I think i did this right, can you check that out? 

okay, that's it! Thanks!

### CODE REVIEW

File: app/layouts/default.njk [hunk 1]
```diff
@@ -1,9 +1,18 @@
 <!DOCTYPE html>
 <html class="{% block pageClass %}{% endblock %}" lang="en">
   <head>
+    <meta property="og:image" content="https://d33wubrfki0l68.cloudfront.net/8afee13b9e497a643b836562965a1a49e580e099/4bab0/images/cover.png">
```
Comment (shyndman) [inline app/layouts/default.njk]: What is this image? It doesn't load for me.

File: app/layouts/default.njk [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+    <meta property="og:image:type" content="image/png">
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://megananderson.design/"/>
+    <meta property="og:title" content="Megan Anderson" />
```
Comment (shyndman) [inline app/layouts/default.njk:8]: Maybe, "Megan Anderson: Product Designer"

**Review by shyndman — CHANGES_REQUESTED** [review]

File: app/layouts/default.njk (hunk 1 above)
Comment (shyndman) [reply app/layouts/default.njk]: I'm not sure where this URL came from, but I think it should be

`/images/cover.png`

Comment (mchristineanderson) [reply app/layouts/default.njk]: yeah it should be that. its just the image that i want displayed when you share a link of the site.

File: app/layouts/default.njk (hunk 2 above)
Comment (mchristineanderson) [reply app/layouts/default.njk:8]: I think since designer is in the description is doesn't need that

Comment (shyndman) [reply app/layouts/default.njk:8]: That makes sense

**Review by shyndman — APPROVED** [review]

**Pull request merged**